config/environments/production.rb

```diff
@@ -1,6 +1,8 @@
 Chessapp::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.default_url_options = { host: 'https://thecheckmates-chessapp.herokuapp.com' }
```

Comment:
hmm, "where our app lives" seems like a piece of information that's not specific to a mailer.

Maybe store the app's URL in a separate variable, and use it wherever necessary?